tests/vmi_configuration_test.go

```diff
@@ -239,6 +239,16 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
 			})
 
+			withMasqueradeBinding := func(vmi *v1.VirtualMachineInstance) {
```

Comment:
How come? Isn't the bridge default?

Reply:
yes but `NewRandomVMIWithEphemeralDisk` which is used in the BeforeEach adds a Masquerade, so no autoattached bridge is added
this is the actual interface type we are using almost everywhere

Reply:
I see it now, thank you!